Dockerfile

```diff
@@ -1,17 +1,14 @@
 # Use an official Python runtime as a parent image
 FROM python:3.11-slim-bookworm
 
+COPY requirements.txt /app_src/requirements.txt
```

Comment:
You do not need to copy a file, as the entire project workspace suppose to be mounted through the docker volume.